app/uk/gov/hmrc/cgtpropertydisposals/repositories/enrolments/VerifiersRepository.scala

```diff
@@ -73,15 +62,16 @@ class DefaultVerifiersRepository @Inject() (mongo: ReactiveMongoComponent)(impli
   override def insert(updateVerifiersRequest: UpdateVerifiersRequest): EitherT[Future, Error, Unit] =
     EitherT[Future, Error, Unit](
       preservingMdc {
-        collection.insert
-          .one[UpdateVerifiersRequest](updateVerifiersRequest)
-          .map[Either[Error, Unit]] { result: WriteResult =>
-            if (result.ok)
+        collection
+          .insertOne(updateVerifiersRequest)
+          .toFuture()
+          .map[Either[Error, Unit]] { result =>
+            if (result.wasAcknowledged())
```

Comment:
ℹ️ This always returns true (with the default write concern). Else is no longer necessary.